pkg/cluster/kubernetes/kubernetes.go

```diff
@@ -106,8 +106,9 @@ type Cluster struct {
 	syncErrors   map[resource.ID]error
 	muSyncErrors sync.RWMutex
 
-	allowedNamespaces map[string]struct{}
-	loggedAllowedNS   map[string]bool // to keep track of whether we've logged a problem with seeing an allowed namespace
+	allowedNamespaces   map[string]struct{}
+	loggedAllowedNS     map[string]bool // to keep track of whether we've logged a problem with seeing an allowed namespace
+	loggedAllowedNSLock *sync.RWMutex
```

Comment:
```suggestion
	loggedAllowedNSLock sync.RWMutex
```

With this you can remove `L131` :-)